apps/typegpu-docs/src/content/examples/tests/tests/index.ts

```diff
@@ -0,0 +1,48 @@
+import tgpu from 'typegpu';
+import * as d from 'typegpu/data';
+import { booleanTests } from './boolean';
+
+const root = await tgpu.init();
+
+const resultBuffer = root.createBuffer(d.i32, 0).$usage('storage');
+const result = resultBuffer.as('mutable');
+
+const computeRunTests = tgpu['~unstable']
+  .computeFn({ in: { num: d.builtin.numWorkgroups }, workgroupSize: [1] })
```

Comment:
No input seems to be used in the function.
```suggestion
  .computeFn({ workgroupSize: [1] })
```